.github/workflows/push-as-image-to-ghcr.yml

```diff
@@ -19,16 +19,40 @@ jobs:
           - coco-as-grpc
           - coco-as-restful
           - rvps
+        verifier:
+          - all-verifier
```

Comment:
`all-verifier` should probably be plural `all-verifiers`, or just `all`?

Reply:
This is a typo for a long time and exists in `verifier` and `attestation-service`. like https://github.com/confidential-containers/trustee/blob/main/deps/verifier/Cargo.toml#L8